src/viem/prepareTransactions.ts

```diff
@@ -16,6 +16,8 @@ import erc20 from 'src/abis/IERC20'
 interface PreparedTransactionsPossible {
   type: 'possible'
   transactions: TransactionRequestCIP42[]
+  feeCurrency: TokenBalance
+  maxGasCostInDecimal: BigNumber
```

Comment:
I think we should have a helper function to get these directly from the prepared transactions, so it remains the single source of truth.
It would probably use `getMaxGasCost` under the hood.
Related: https://github.com/valora-inc/wallet/pull/4415#discussion_r1380576424

@cajubelt did that already in https://github.com/valora-inc/wallet/pull/4415/files#diff-0412194130424f92cdf96a29da9526a23b02f6ab75c78a899bc934e9c8f1ba08R102

Reply:
Hum, maybe it's fine to attach (some of ?) these to `PreparedTransactionsResults` after all 🤔